canvas_modules/common-canvas/src/common-canvas/common-canvas.scss

```diff
@@ -199,5 +183,44 @@ $bottom-panel-height: 393px;
 	&.info > svg > .dot {
 		fill: $support-04;
 	}
+}
+
+.bottom-panel {
+	display: flex;
+	flex-direction: column;
+	height: 50vh;
+	position: fixed;
+	bottom: 0;
+}
+
+.top-panel {
+	display: flex;
+	flex: 0 0 50%;
+	flex-direction: row;
+}
+
+.bottom-panel-drag {
+	background: transparent;
+	cursor: row-resize;
+	flex: 0 0 2px;
+	transition: all 0.2s ease-in;
+	&:hover {
+		background: $interactive-01;
+	}
+}
 
+.bottom-panel-contents {
+	background-color: #FFFFFF;
+	flex: 1;
+	width: 1510px;
+	margin: auto;
+	user-select: none;
+}
```

Comment:
I don't know why `background-color: #FFFFFF;` is set.  This should be up to the host application to set.  Why is `width` hardcoded?

Reply:
Good catch @matthoward366   We set the default background-color of the palette and right-flyout panel to `$ui-01` , which the host app can override if it wishes, so I suggest you, Nabil, change it to that.

As for the width, that is working OK in the current code in master where the width of the bottom panel will automatically adjust depending on the size of the palette and size of the right-side flyout (if present). We need to make it so that behavior remains.